BuildingBlocks/src/BuildingBlocks.API/Serilog/BackboneIdMaskingOperator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Serilog.Enrichers.Sensitive;
+
+namespace Backbone.BuildingBlocks.API.Serilog;
+public class BackboneIdMaskingOperator : RegexMaskingOperator
```

Comment:
Maybe we should call this class "StronglyTypedIdMaskingOperator". We use that name (StronglyTypedId) in several places already. BackboneId on the other hand is a new name for an existing concept.

Reply:
Done, I like that, I actually struggled to think of a name for that one.